src/AVIMetaEditBundle/Resources/views/Download/macTable.html.twig

```diff
@@ -1,10 +1,10 @@
 <tr>
-    <th rowspan="2" colspan="2"><a href="https://itunes.apple.com/app/avimetaedit/id1289778020">Mac App Store</a></th>
+    <th rowspan="2" colspan="2"><a href="https://www.apple.com/itunes/affiliates/download/?id=510620098">Mac App Store</a></th>
```

Comment:
Even if the link is named "Mac App Store", I think I prefer here the link to the product (the geo link of each app, for each app).

Reply:
Done.